internal/orchestrator/helpers.go

```diff
@@ -93,6 +94,10 @@ func parseAppStatus(containers []container.Summary) []AppStatusInfo {
 			appendResult(appPath, StatusStopping)
 			continue
 		}
+		if slices.ContainsFunc(s, func(v Status) bool { return v == StatusStopped }) {
```

Comment:
we should have some test for checking the whole app transicition, I want to avoid that if a stop an application this transitioning in the fail state. So this check is ok, if all the other container are running, but if I am stopping an app, I don't want to get a fail state during the transition from running to stopped